components/scream/src/share/io/tests/io.cpp

```diff
@@ -34,6 +38,77 @@ std::shared_ptr<UserProvidedGridsManager>   get_test_gm(const ekat::Comm& io_com
 ekat::ParameterList                         get_om_params(const Int casenum, const ekat::Comm& comm);
 ekat::ParameterList                         get_in_params(const std::string type, const ekat::Comm& comm);
 
+// TEST_CASE("ciccio")
```

Comment:
This is commented out, should we delete, or is this a TODO for the next PR?

Reply:
Oh, lol! I added that while debugging, and completely forgot. I am thankful I didn't follow my usual habit of filling debug messages/vars/names with "colorful" italian words... :D

Reply:
Your debug messages are helping me prepare for the next time I watch soccer with my in-laws...